src/main/webapp/app/course/overview/course-overview/course-overview.component.scss

```
@@ -1 +1,23 @@
+.layout-content {
+    position: relative;
+}
+
+// For tabs that opt in, the shared action bar floats over the top of the content column, offset past
+// the secondary sidebar, so the sidebar itself reaches the very top. It slides to full width when the
+// sidebar collapses, reusing the same 0.1s timing as the sidebar-width transition (no vertical jump).
+jhi-course-title-bar.title-bar-over-content {
+    position: absolute;
+    top: 0;
+    right: 0;
+    // 263px == secondary sidebar width (255px) + 8px gap, matching .content-sidebar-width.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)